### PR TITLE
Issues/4567 zentao bp

### LINKED ISSUE #4567: [Feature][Zentao] Support Config UI blueprint
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar feature requirement.

### Use case

I want to add Zentao data to my project, which is not available in v0.16

### Description

TODO:

- [ ]  Backend: support BPv200 for Zentao
- [ ]  Frontend: support BP normal mode

### Related issues

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by likyh**
### Summary
1. add zentao bp
2. convert execution to sprint

### Does this close any open issues?
Closes #4567 

### Screenshots
image
image
image

### CODE REVIEW

**Review by warren830 — APPROVED** [review]
TL,DR

File: config-ui/src/plugins/components/data-scope-list/use-data-scope-list.ts [hunk 1]
```diff
@@ -70,6 +70,12 @@ export const useDataScopeList = ({ plugin, connectionId, scopeIds }: UseDataScop
             name: sc.name,
             transformationRuleName: sc.transformationRuleName,
           };
+        case plugin === 'zentao':
+          return {
+            id: sc.id,
+            name: sc.name,
+            transformationRuleName: sc.transformationRuleName,
```
Comment (mintsweet) [inline config-ui/src/plugins/components/data-scope-list/use-data-scope-list.ts:77]: I see no transformation in zentao above, why add it here?

Comment (likyh) [reply config-ui/src/plugins/components/data-scope-list/use-data-scope-list.ts]: Yes, That's unnecessary. Fixed

**Review by mintsweet — APPROVED** [review]
LGTM.

**Review by warren830 — APPROVED** [review]

**Pull request merged**